src/routes/boot.ts

```diff
@@ -477,7 +493,10 @@ const anonymousBoot = async (
       shouldVerify,
     },
     visit,
-    alerts: { ...ALERTS_DEFAULT, changelog: false },
+    alerts: {
+      ...excludeProperties(ALERTS_DEFAULT, ['lastFeedSettingsFeedback']),
```

Comment:
Wait did you want to exclude all here?

Reply:
The function would only exclude the second parameter, which is the new property. This is just to make it consistent on what is being returned.